docs/application/native/guides/text-input/tts.md

```diff
@@ -327,6 +360,22 @@ To obtain the current state, the supported voice list, and the current voice:
     }
     ```
 
+- Get the current service state using the `tts_get_service_state()` function.
+
```

Comment:
Please add more descriptions of service state in ## TTS information retrieval.

Reply:
Thank you for your opinion.
I fix the guide as similar as csharp one.